python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/class_maps.schema.yml

```
@@ -59,3 +59,14 @@ keys:
                   type: str
                   description: IPv6 Access-List Name.
                   convert_types: [ int ]
+            dscp:
+              type: dict
+              description: It allows match on either DSCP, ECN or both for the IPV4 and IPV6 packets.
+              keys:
+                dscp_value:
```

This key should be required
```suggestion
                dscp_value:
                  required: true
```

---

Added!